tests/spread/smoketests/init-fast-checks/task.yaml

```
@@ -0,0 +1,26 @@
+summary: Check that the templates pass their own linting and unit tests.
+
+# Limited to Ubuntu 22.04 and 24.04 because the templates require Python >= 3.10.
+systems:
+  - ubuntu-22.04-64
+  - ubuntu-24.04-64
```

```suggestion
systems:
  - ubuntu-24.04-64
```

Hmmm thinking about this further, how about we only do it on 24.04? We don't really need to support older Ubuntu releases than that, and this will save us hassle when we set 24.04 as the default base.

---

Sounds good, especially as we're not packing or deploying the charms. I've just pushed a commit for this.